readthedocs/projects/views/private.py

```diff
@@ -191,14 +209,28 @@ def project_version_detail(request, project_slug, version_slug):
                 )
                 version.built = False
                 version.save()
-        url = reverse('project_version_list', args=[project.slug])
-        return HttpResponseRedirect(url)
+        return HttpResponseRedirect(self.get_success_url())
 
-    return render(
-        request,
-        'projects/project_version_detail.html',
-        {'form': form, 'project': project, 'version': version},
-    )
+
+class ProjectVersionDeleteHTML(ProjectVersionMixin, GenericModelView):
+
+    http_method_names = ['get']
```

Comment:
Isn't this the same case as in another refactor PR where GET and POST do the exact same thing? If so, don't we should list it here and make POST to call `.get`?

Reply:
I didn't find any place where we call it form a post method, but I'm adding it just in case. This is also on the list https://github.com/readthedocs/readthedocs.org/issues/6189